modules/datepicker/datepicker-ios.tsx

```diff
@@ -4,14 +4,25 @@ import {BaseDateTimePicker} from './basepicker'
 import {IosDatetimePickerProps} from './types'
 
 export const DatePicker = (props: IosDatetimePickerProps): JSX.Element => {
+	/**
+	 * Allows us to override the iOS picker's button's minwidth
+	 * which is set below to 120 to accomodate the hours report
+	 * editor which changes minwidth to create a compact look.
+	 */
+	let baseStyle = {
+		...defaultStyle.datePicker,
+		...props.style,
+	}
+
 	return (
 		<BaseDateTimePicker
+			displayIos={props.display}
 			initialDate={props.initialDate}
 			minuteInterval={props.minuteInterval}
 			mode={props.mode}
 			onDateChange={props.onDateChange}
 			showPickerIos={true}
-			style={defaultStyle.datePicker}
+			style={baseStyle}
```

Comment:
```suggestion
			style={[defaultStyle.datePicker, props.style]}
```

Reply:
I think inlining it is a good suggestion. Types are a picking up on how `props.style` can be undefined, though. Is there a fallback we can provide?

Reply:
Is it because it's set to `style?: ViewStyle` instead of `style?: StyleProp<ViewStyle>`?

Reply:
You guessed right. And it was my mistake months ago 😅